src/pages/Donate/DonateDialog.tsx

```diff
@@ -6,29 +6,36 @@ import {
   TextField,
   Typography,
 } from '@mui/material';
-import { useDonate } from '../../api/donate';
+import { UseMutationResult } from '@tanstack/react-query';
+import { AxiosError } from 'axios';
 import React, { useEffect, useState } from 'react';
-import { GetUserDetailsResponse } from '../../types/UserTypes';
-import SpinningButton from '../../components/SpinningButton';
 import Avatar from '../../components/Avatar';
+import SpinningButton from '../../components/SpinningButton';
+import { GetUserDetailsResponse } from '../../types/UserTypes';
 import { getErrorMessage, valueAsNumber } from '../../utils/utils';
 
 interface DonateDialogProps {
   creator: GetUserDetailsResponse;
   closeDialog: () => void;
+  mutation: UseMutationResult<void, AxiosError<unknown, any>, number, unknown>;
```

Comment:
tak jak robimy przy wywoływaniu `useMutation`:
```suggestion
  mutation: UseMutationResult<void, AxiosError, number>;
```

Reply:
ok